.github/workflows/pull-request.yml

```
@@ -544,3 +544,123 @@ jobs:
+          else
+            echo "❌ Seeding failed - Debug Information:"
+            echo "Container status:"
+            docker ps | grep talawa
+            echo "Recent container logs:"
+            docker logs talawa-api-1 --tail 50
+            echo "=== Users table contents ==="
+            docker exec talawa-postgres-1 psql -U talawa -d talawa \
+            -c "SELECT id, email_address, name, role FROM users;" 2>/dev/null || echo "Could not query users"
+            exit 1
+          fi
+                    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)